### DISCUSSION TITLE
no UI option showed.

### DISCUSSION THREAD

**Discussion by L0bit0** [totaldebug/atomic-calendar-revive#964] [Support] [answered]
Hi

I just installed, and first card I have created show UI options, but required section did not showed.

I edited YAML to add some calendar entities and UI options are not showed anymore.

i tried to restart homeassistant, but stil not showing:
![image](https://user-images.githubusercontent.com/10359623/222923981-e0ecc186-121e-459f-8506-5d15ba8fc2a9.png)

Any Idea?

thanks!

**Comment by marksie1988** [comment]
Please share the yaml config that you have entered, along with what version of home assistant you are using.

**Comment by L0bit0** [comment, marked as answer]
Hi, sorry for the last response, i was "out of order" some days :-)

Now I can see... I assume last Home Assistant updates fixed...

This is my code anyway:

> type: custom:atomic-calendar-revive
> name: Calendar Card
> entities:
>   - calendar.dani_del_amo_gmail_com
>   - calendar.danialmu
>   - calendar.deva
>   - calendar.nora
>   - calendar.linarespac_gmail_com
> enableModeChange: true
> firstDayOfWeek: 1
> maxDaysToShow: 7
> refreshInterval: 60
> defaultMode: calendar

Thanks!
